dapp/src/components/TransactionModal/ActiveUnstakingStep/UnstakeFormModal/UnstakeDetails.tsx

```diff
@@ -31,9 +24,6 @@ function UnstakeDetails({
 
   return (
     <Flex flexDirection="column" gap={10} mt={10}>
```

Comment:
It looks like we can get rid of this wrapper.  So let's add `mt` to the `List` component.

```
<List spacing={3} mt={10}>
 ...
</List>
```

Reply:
Ref commit: 8ccb337b